pragma-entities/src/error.rs

```diff
@@ -85,3 +87,19 @@ impl Error for InteractError {
         InfraError::InternalServerError
     }
 }
+
+#[derive(Debug, thiserror::Error, ToSchema)]
+pub enum RedisError {
```

Comment:
why do we have twice the same error type?

Reply:
That was to mimic our current model of having `InfraError` -> `HandlerError`